lib/turn.rb

```diff
@@ -0,0 +1,36 @@
+class Turn
+
+  attr_reader :guess, :card, :turn
+
+  def initialize(guess, card)
+    #@turn = []
+    @card = card
+    #@turn_counter = []
+    @guess = guess
+  end
+
+  def correct?
+     @guess == card.answer
```

Comment:
Again, be careful about referencing instance variables (like `@card`) without the `@` sign.